tests/api/test_account.py

```diff
@@ -414,3 +415,148 @@ def test_set_password(user_api_client, customer_user):
 
     customer_user.refresh_from_db()
     assert customer_user.check_password(password)
+
+
+@patch('saleor.account.emails.send_password_reset_email.delay')
+def test_create_user_password_reset_email(
```

Comment:
We already have a test for the `customerCreate` mutation.  I think we can extend it with the additional variable `send_mail: True` and test email sending there.